infrastructure/tests/test_load_data.py

```diff
@@ -95,18 +98,23 @@ def test_check_file(self):
             assert False
 
     def test_load_file_with_bad_data(self):
-        fp = self.generate_bad_data()
-
-        self.assertRaises(ValueError, utils.load_file, ImportCSVTestCase.geography, fp)
+        try:
+            fp = self.generate_bad_data()
+        except:
+            raise ValueError(
+                "Failed to generate bad data: %s"
+                % fp
+            )
+            #self.assertRaises(ValueError, utils.load_file, ImportCSVTestCase.geography, fp)
```

Comment:
I don't understand why we're changing this - might be good to revert and understand better or drop